src/components/your-services/index.njk

```diff
@@ -55,10 +55,23 @@
           {% set locale = ['clientRegistry.', env, '.', service.client_id, '.'] | join %}
           <li class="your-services__card">
             <div class="your-services__card__content">
-            <{{serviceHeadingLevel}} class="govuk-heading-s govuk-!-font-weight-regular" data-test-id="service-card-heading">
-              <a class="govuk-link" href="{{ [locale, 'link_href'] | join | translate }}">{{ [locale, 'link_text'] | join | translate }}</a>
-            </{{serviceHeadingLevel}}>
-            <p class="govuk-body your-services__card__last-used">{{ 'pages.yourServices.lastUsed' | translate }}: {{ service.last_accessed_readable_format }}</p>
+            {% if service.isHMRC %}
```

Comment:
do we need this? can we not just check if there is a `paragraph1` field? Would make it more semantic